Planner/Planner/src/Users.java

```diff
@@ -0,0 +1,47 @@
+import javax.swing.JFrame;
+import javax.swing.JButton;
+import javax.swing.JPanel;
+import javax.swing.JList;
+import javax.swing.JOptionPane;
+
+import java.awt.FlowLayout;
+import java.awt.Frame;
+import java.awt.event.ActionEvent;
+import java.awt.event.ActionListener;
+
+public class Users extends JFrame implements ActionListener {
+
+	public Users() {
+		super ("Planner");
+		setLayout(new FlowLayout());
+		setSize(800, 600);
+		setLocation(200, 100);
+		setDefaultCloseOperation(JFrame.EXIT_ON_CLOSE);
+		setVisible(true);
+		
+		JPanel panel = new JPanel(new FlowLayout(FlowLayout.RIGHT));
+		JButton user = new JButton("Add new User");
+		JButton task = new JButton("Add new Task");
+		String[] userList = {"Justyna", "Piter", "Kris", "Basia"};
+		String[] taskList = {"Wash clothes", "Shopping", "Wash a car", "Prepare breakfast", 
```

Comment:
Here I would like to see some ArrayList<User> and ArrayList<Task>, array lists of User and Task objects, not of strings.